docs/extensions/get-started/anatomy.md

```diff
@@ -0,0 +1,98 @@
+# Extension Anatomy
+
+You've now learnt how to get basic extension running. In this topic you will learn some fundamental concepts to Lens Extension development; How does it work under the hood?
+
+`Hello World` extension does three things:
+
+- Hooks on `onActivate()` and ouputs message into console.
+- Hooks on `onDectivate()` and ouputs message into console.
+- Registers `ClusterPage` so that page is visible in the sidebars of cluster dashboards.
+
+Let's take a closer look at Hello World sample's source code and see how to achieve these things.
+
+## Extension File Structure
+
+```
+.
+├── .gitignore          // Ignore build output and node_modules
+├── Makefile            // Config for build tasks that compiles the extension
+├── README.md           // Readable description of your extension's functionality
+├── src
+│   └── page.tsx         // Extension's additional source code
+├── main.ts             // Source code for extension's main entrypoint
+├── package.json         // Extension manifest and dependencies
+├── renderer.tsx         // Source code for extension's renderer entrypoint
+├── tsconfig.json        // TypeScript configuration
+├── webpack.config.json  // Webpack configuration
```

Comment:
`webpack.config.js` or `.ts` (in case of using `ts-node`)

Reply:
> `webpack.config.js` or `.ts` (in case of using `ts-node`)

good catch. Will fix it